### ISSUE TITLE
Colour of errors

### ISSUE THREAD

**Issue by TotalVerb** [tonyhffong/Lint.jl#143]
I feel like the choice of magenta as the colour for errors is unnatural. I would personally have chosen red.

I don't have a very strong opinion about this however, and it would be nice to know what people generally prefer.

**Comment by Michael-Klassen** [comment]
I personally like magenta, but red is good too. What does @tonyhffong prefer?

**Comment by tonyhffong** [comment]
red is slightly unpleasant on my retinas. Just personal opinion.

**Comment by TotalVerb** [comment]
I think that the best option is to use the Julia default colors. These can be changed in [Julia itself](https://docs.julialang.org/en/latest/manual/interacting-with-julia#Customizing-Colors-1) now, so the issue of red being unpleasant can be solved in the Julia configuration itself.

**Comment by musm** [comment]
I still think it's odd and not natural to use a nonstandard color such as magenta to visually highlight errors

**Comment by TotalVerb** [comment]
Right; this is why in #210 I propose to use the Julia default colors, which will default to red for errors, which can be customized in `.juliarc`.

**Comment by musm** [comment]
perfect!
